sdk/python/src/dagger/api/base.py

```diff
@@ -136,10 +131,15 @@ def _structure_response(
 
 
 class Arg(NamedTuple):
+    py_name: str
     name: str
     value: Any
+    type_: type
     default: Any = attr.NOTHING
 
+    def is_valid(self) -> bool:
+        return is_bearable(self.value, self.type_)
```

Comment:
Not an issue, but just don't understand the naming here `is_bearable` ?

Reply:
It's just the API of the library I'm using: [beartype](https://pypi.org/project/beartype/). They use a lot of "bear" word plays like that. `is_bearable` means _is it valid_?